industry_benchmarks/utils/data_gathering.py

```diff
@@ -1,18 +1,39 @@
 import click
+from collections import defaultdict
 import pathlib
 import json
-import abc
 import rdkit
+import tqdm
 from rdkit import Chem
 from rdkit.Chem import AllChem
 from rdkit.Chem import rdFreeSASA
 import gufe
 from gufe import SmallMoleculeComponent, LigandAtomMapping, AtomMapping
+from gufe.tokenization import JSON_HANDLER
 import openfe
 from openfe import LigandNetwork
 from kartograf.atom_mapping_scorer import (
     MappingRMSDScorer, MappingShapeOverlapScorer, MappingVolumeRatioScorer,
 )
+import shutil
+from openff.units import unit
+
+# define all the result files we want to collect
+RESULT_FILES = [
+    # data files
+    "structural_analysis_data.npz",
+    "energy_replica_state.npz",
+    "simulation_real_time_analysis.yaml",
+    "info.yaml",
+    # png analysis files,
+    "forward_reverse_convergence.png",
+    "ligand_COM_drift.png",
+    "ligand_RMSD.png",
+    "mbar_overlap_matrix.png",
+    "protein_2D_RMSD.png",
+    "replica_exchange_matrix.png",
+    "replica_state_timeseries.png"
+]
```

Comment:
So there's two aspects to this that we want to consider:

1. How do we make accessing these files easy for us. I.e. would storing relative paths of "where are these files" make it easier for us to postprocess the data using some kind of data analysis script?
2. Making a manifest of the data being stored. This one is harder to predict exactly what folks will need. Depending on how much effort it is for us, it might be easier to just store a list of all the files and their paths and dump that out. That being said, if it's just a pain then maybe just the relative path of "here is where the files are" is good enough.

Reply:
1. Currently I have the files stored in folders that have the same naming as the edges in the `all_network_properties.json` file, so an analysis script which loops over the edges in the json should be able to load the raw results from the local folder with the same name. See this tree of the zip archive
```
├── all_network_properties.json
├── complex_spiro2_spiro1_repeat_0
│   ├── energy_replica_state.npz
│   ├── info.yaml
│   ├── simulation_real_time_analysis.yaml
│   └── structural_analysis_data.npz
├── complex_spiro2_spiro1_repeat_1
│   ├── energy_replica_state.npz
│   ├── info.yaml
│   ├── simulation_real_time_analysis.yaml
│   └── structural_analysis_data.npz
├── complex_spiro2_spiro1_repeat_2
│   ├── energy_replica_state.npz
│   ├── info.yaml
│   ├── simulation_real_time_analysis.yaml
│   └── structural_analysis_data.npz
├── solvent_spiro2_spiro1_repeat_0
│   ├── energy_replica_state.npz
│   ├── info.yaml
│   ├── simulation_real_time_analysis.yaml
│   └── structural_analysis_data.npz
├── solvent_spiro2_spiro1_repeat_1
│   ├── energy_replica_state.npz
│   ├── info.yaml
│   ├── simulation_real_time_analysis.yaml
│   └── structural_analysis_data.npz
└── solvent_spiro2_spiro1_repeat_2
    ├── energy_replica_state.npz
    ├── info.yaml
    ├── simulation_real_time_analysis.yaml
    └── structural_analysis_data.npz

```
2. A manifest might make everyone more confident in what we are storing, would the above tree be enough with a general description of what is in each file?

Reply:
Just to me that works, thanks!